pkg/core/proxy/integrations/http/util.go

```diff
@@ -427,6 +427,37 @@ func isPassThrough(logger *zap.Logger, req *http.Request, destPort uint, opts mo
 				continue
 			}
 		}
+		if bypass.URLMethods != nil && len(bypass.URLMethods) != 0 {
+			for _, method := range bypass.URLMethods {
+				if method == req.Method {
+					passThrough = true
```

Comment:
two cases~
path is api/users and mehtod is post
path is api/data and method is post

if I pass the param of POST in the bypass rule wont' both yield me the same result?